<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>history_data</class>
 <widget class="QWidget" name="history_data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1166</width>
    <height>673</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>检测数据查询</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resourse/chaxun.png</normaloff>:/resourse/chaxun.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QToolBar" name="ToolBar_action">
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <addaction name="action_return"/>
     <addaction name="action_detail"/>
     <addaction name="action_delete"/>
     <addaction name="separator"/>
     <addaction name="action"/>
     <addaction name="action_find"/>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableView" name="TV_ShowData">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="action_return">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resourse/laststep.png</normaloff>:/resourse/laststep.png</iconset>
   </property>
   <property name="text">
    <string>返回上一级</string>
   </property>
   <property name="toolTip">
    <string>返回上一界面</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="action_detail">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resourse/nextstep.png</normaloff>:/resourse/nextstep.png</iconset>
   </property>
   <property name="text">
    <string>查看明细</string>
   </property>
   <property name="toolTip">
    <string>查看该条记录的测试明细</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resourse/ic_share_line.png</normaloff>:/resourse/ic_share_line.png</iconset>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
   <property name="toolTip">
    <string>导出到Excel</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="action_delete">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resourse/delete (1).png</normaloff>:/resourse/delete (1).png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除选中测试记录</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="action_find">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resourse/find.png</normaloff>:/resourse/find.png</iconset>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
   <property name="toolTip">
    <string>根据生产批次号等信息查找生产批次</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
